modules/java/test/android_test/tests_module/build.gradle.in

```diff
@@ -0,0 +1,35 @@
+apply plugin: 'com.android.application'
+
+android {
+    namespace 'org.opencv.tests'
+    compileSdkVersion @ANDROID_COMPILE_SDK_VERSION@
+    defaultConfig {
+        applicationId "org.opencv.tests"
+        minSdkVersion @ANDROID_MIN_SDK_VERSION@
+        targetSdkVersion @ANDROID_TARGET_SDK_VERSION@
+        versionCode 301
+        versionName "3.01"
```

Comment:
> version

is it correct?

Reply:
Not sure, but it's the same everywhere. Looks like we need to substitute OpenCV version here and in samples.

Reply:
Let's do it in a separate PR.